myapp/app/models/task.rb

```diff
@@ -1,2 +1,4 @@
 class Task < ApplicationRecord
+    enum priority: { low: 0, medium: 1, high: 2 }  
+    enum status:   { "to do": 0, "in progress": 1, "done": 2 }
```

Comment:
There is incorrect indentation and trailing whitespace.
```suggestion
  enum priority: { low: 0, medium: 1, high: 2 }
  enum status:   { "to do": 0, "in progress": 1, "done": 2 }
```